site/content/blogs/release-v132.ja.md

```diff
@@ -0,0 +1,60 @@
+---
+title: 'AWS Copilot v1.32: `run local --proxy`, `run local --watch`, 既存 ALB のインポートをサポート'
```

Comment:
Sorry for nitpicking: please use Japanese comma "、" here
cf) https://github.com/aws/copilot-cli/blob/mainline/site/content/blogs/release-v130.ja.md?plain=1#L10